### PR TITLE
data:aadityadike

### PR DESCRIPTION
**Pull request by aadityadike**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4330"></a>

### CODE REVIEW

**Review by rahulpandey70 — APPROVED** [review]
Looks good

**Review by arpitghura — APPROVED** [review]
looks good

**Review by rupali-codes — CHANGES_REQUESTED** [review]
I think under `links` array, `Name` should be `name` and should not start with a `capital` letter

**Comment by Pradumnasaraf** [comment]
> I think under `links` array, `Name` should be `name` and should not start with a `capital` letter

Great eye, Rupali. You are 100% correct.

File: data/aadityadike.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        "Open Source"
+    ],
+    "links": [
+        {
+            "Name":"Twitter",
+            "icon": "FaTwitter",
+            "url": "https://twitter.com/adityastwt1"
+        },
+        {
+            "Name":"Github",
```
Comment (Pradumnasaraf) [inline data/aadityadike.json:19]:
```suggestion
            "name":"Github",
```

File: data/aadityadike.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            "icon": "FaGithub",
+            "url": "https://github.com/aadityadike"
+        },
+        {
+            "Name":"LinkedIn",
```
Comment (Pradumnasaraf) [inline data/aadityadike.json:24]:
```suggestion
            "name":"LinkedIn",
```

File: data/aadityadike.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            "icon": "FaLinkedin",
+            "url": "https://www.linkedin.com/in/aditya-dike-72837a221"
+        },
+        {
+            "Name":"HashNode",
```
Comment (Pradumnasaraf) [inline data/aadityadike.json:29]:
```suggestion
            "name":"HashNode",
```

File: data/aadityadike.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    "bio": "Full-Stack Development | DevOps | Open Source",
+    "tags": [
+        "Java",
+        "Javascript",
+        "Docker",
+        "Open Source"
+    ],
+    "links": [
+        {
+            "Name":"Twitter",
```
Comment (Pradumnasaraf) [inline data/aadityadike.json:14]:
```suggestion
            "name":"Twitter",
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey

The `name:` key in the links shout start will small `n`

**Pull request closed without merging**